docs/configuration-v1_3-exp.md

```diff
@@ -190,6 +190,16 @@ The Fedora CoreOS configuration is a YAML document conforming to the following s
     * **_gid_** (integer): the group ID of the new group.
     * **_password_hash_** (string): the hashed password of the new group.
     * **_system_** (bool): whether or not the group should be a system group. This only has an effect if the group doesn't exist yet.
+* **_boot_device_** (object): describes the desired boot device configuration.
+  * **_layout_** (string): the disk layout of the target OS image. This is only needed if also specifying `luks` or `mirror`. Supported values are `aarch64`, `ppc64le`, and `x86_64`. Defaults to `x86_64`.
```

Comment:
> That makes it seems like there's a reason for using boot_device without either of those, but there isn't, right?

That's exactly the problem I'm trying to avoid: users thinking they need to specify the `layout` if they're not specifying `luks` or `mirror`.  Your approach seems better for that.  (There's not really a way to validate it, though, since we don't use struct pointers generally.)